CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+Contributing to OpenJFX 11 Updates
+==================================
+
+The OpenJFX 11 Updates repository is _not_ accepting new contributions. If you are looking to contribute to OpenJFX, please visit [openjdk/jfx](https://github.com/openjdk/jfx) instead.
+
+Important bug fixes that have already been fixed in the mainline can be backported with approval. New fixes should first go into the mainline, and then can be considered for backporting. Approval from one of the project leads is required prior to integrating the fix.
+
+To backport a fix, you need to import the patch from mainline into a branch of your personal fork of this repo. Then, you can create the pull request. We strongly recommend creating a backport-style pull request, by creating the PR with the title:
+
```

Comment:
shouldn't this be a requirement instead of a recommendation? I don't think there is a reason why using that style would not be possible?

Reply:
For fixes that are backports it should always be possible, so I'll reword this to make that clear.

Reply:
Updated to make it clear that backport PRs are needed.